ci-operator/step-registry/openshift-qe/conc-builds/openshift-qe-conc-builds-ref.yaml

```diff
@@ -1,9 +1,7 @@
 ref:
   as: openshift-qe-conc-builds
-  from_image:
-    namespace: ci
-    name: ocp-qe-perfscale-ci
-    tag: latest
+  from: ocp-qe-perfscale-ci
+  cli: latest
```

Comment:
yes, in all the tests I've been running using[ Dockerfile_test](https://github.com/openshift-qe/ocp-qe-perfscale-ci/blob/main/prow/Dockerfile_test) that doesnt have the oc in the dockerfile. But this will effect all tests so trying to figure out the scope first